### ISSUE TITLE
Functions limitation

### ISSUE THREAD

**Issue by suntong** [lunet-io/scriban#13]
For the following code,

```
            var template = Template.Parse(@"This {{ ""is"" | string.upcase }} {{ text | string.capitalize }} ({{ text }}),{{""\n""}} and {{myfunction}} from scriban!");
            var model = new { text = "Hello Text" };
            var scriptObject = new ScriptObject();
            scriptObject.Import(model);
            // Import the following delegate to scriptObject.myfunction (would be accessible as a global function)
            scriptObject.Import("myfunction", new Func<string>(() => "Hello Func"));

            var context = new TemplateContext();
            context.PushGlobal(scriptObject);
            template.Render(context);
            context.PopGlobal();
```

The output is, 

```
This IS  (Hello Text),
 and Hello Func from scriban!
```

I.e., why `{{ text | string.capitalize }}` is not working as I exptected, while both `{{ ""is"" | string.upcase }}` and `{{ text }}` are just fine?

**Comment by xoofx** [comment]
This should be fixed by  	3625d70. The nuget 0.2.1 is on its way.

**Comment by suntong** [comment]
Having upgraded to nuget **0.2.1**, I am suddenly getting the following errors for the code that was building fine:

Error	The type or namespace name 'Scriban' could not be found (are you missing a using directive or an assembly reference?)

Is it because my project is on .NET Framework 4.5, and need to be upgraded to some higher version?
